docs/src/getting_started.md

```diff
@@ -26,7 +26,14 @@ It doesn't take much to get `stata_kernel` up and running. Here's how:
 
 ## Package Install
 
-To install the kernel, from a terminal or command prompt run:
+If you use Anaconda or Miniconda, from the Anaconda Prompt run:
+
+```bash
+conda install -c conda-forge stata_kernel
+python -m stata_kernel.install
```

Comment:
probably good to suggest `python3` as below

Reply:
AFAIK for Anaconda/Miniconda distributions the name of the executable is always `python`, regardless of OS or other versions present. When someone runs it from the anaconda prompt it should always find the correct python executable. Added pip3 to the later instructions for installing notebook/jupyterlab, though.

Reply:
With Anaconda, `python` and `python3` are both symlinks to the latest release:
```
> ll ~/local/anaconda3/bin/python*
lrwxr-xr-x  1 kyle  staff     9B Jun  8 16:08 /Users/kyle/local/anaconda3/bin/python -> python3.7
lrwxr-xr-x  1 kyle  staff     9B Jun  8 16:08 /Users/kyle/local/anaconda3/bin/python3 -> python3.7
-rwxrwxr-x  1 kyle  staff   4.0M Jun  8 16:07 /Users/kyle/local/anaconda3/bin/python3.7
```

I agree that `python3` isn't usually necessary, but I'm trying to think about the user who's new to Python, and especially mac users whose default `python` points to the `/usr/bin/python` 2.7 might get confused.

Reply:
Ah, I did not now this (I use Windows and there is no python3 symlink there). Will do the change.